runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -253,4 +343,4 @@ private boolean isInCrypt()
 		Player localPlayer = client.getLocalPlayer();
 		return localPlayer != null && localPlayer.getWorldLocation().getRegionID() == CRYPT_REGION_ID;
 	}
-}
+}
```

Comment:
Re-add the newline at EOF
```suggestion
}

```